framework/grid/Column.php

```diff
@@ -1,4 +1,5 @@
 <?php
+
```

Comment:
Is the newline required by PSR?

Reply:
yes, `PSR12.Files.FileHeader.SpacingAfterBlock`